tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -975,6 +975,12 @@ show_techsupport/test_techsupport.py::test_techsupport:
 #######################################
 #####            snmp             #####
 #######################################
+snmp/test_snmp_memory.py::test_snmp_memory_load:
+  skip:
```

Comment:
>skip [](http://example.com/codeflow?start=2&length=4)

The purpose of this PR is to skip a testcase based on github issue. Let's only do one thing.

If you change test case at the same time, it is not tested in PR checker.

Reply:
we are skipping test_snmp_memory_load - this check for snmp free memory OID.

I have added a new test test_snmp_avail_memory_load - this is to check the available memory based on memUsage OID.
This test will also increase memory load using the same load_memory() method

Reply:
Split the diff, created a separate PR to skip test_snmp_memoy_load test: https://github.com/sonic-net/sonic-mgmt/pull/9034